src/ch/imedias/rsccfx/view/RsccSupportView.java

```diff
@@ -128,6 +124,9 @@ private void layoutForm() {
     VBox.setVgrow(startServiceInnerPane, Priority.ALWAYS);
     startServiceInnerPane.getStyleClass().add("contentSupport");
 
+    validationImgView.setId("validationImgView");
```

Comment:
Same as my comment in CSS. If you don't plan to use the CSS, you can delete this file. If you think that consistency needs to be applied, leave it in but write a comment in the css.